cohort_4/playbook/ACTIVITIES.md

```diff
@@ -88,6 +89,109 @@
 
 <!-- markdown-toc end -->
 
+# Creating a safe space
+
+The one rule that we have at Hack Camp is that we make it a safe space.
+We all know how bad it feels when we are made fun of or when someone makes
+us feel uncomfortable.
+
+Below are some scenarios which may occur and how to handle them:
```

Comment:
Cool, so just to clarify, you are suggesting to remove:
> ## Someone writes or says "X person is gay." in a playful joking manner

Correct? If So I say just keep it for now. I feel that we will add more things to it later and it would be one less thing I need to change as of now?